rawdisk/scheme/mbr.py

```diff
@@ -47,6 +56,43 @@ def part_offset(self):
     def part_type(self):
         return self.fields.part_type
 
+    def chs2lba(self, cyl, head, sect, geometry=DEFAULT_GEOMETRY):
+        """ helper for consistency check """
+        hpc, spt = geometry['HPC'], geometry['SPT']
+        return sect-1 + head*spt + cyl*hpc*spt
+
+    def get_type_label(self):
+        if self.fields.part_type in TYPES:
+            return '%s' % TYPES[self.fields.part_type]
+        else:
+            return 'unknown'
+
+    def __str__(self):
+        # calculated values
+        fields = self.fields.copy()
+        fields['chs_start_sector'] = self.chs2lba(
```

Comment:
Because `chs_start_sector` is not `self.fields.starting_sector` and may confuse people. Do you know a better name?